packages/playwright-test/src/index.ts

```diff
@@ -253,6 +253,8 @@ export const test = _baseTest.extend<TestFixtures, WorkerFixtures>({
     if (debugMode())
       testInfo.setTimeout(0);
 
+    const screenshotOptions = typeof screenshot !== 'string' ? screenshot : undefined;
```

Comment:
To avoid unexpected properties being passed to `page.screenshot`:

```suggestion
    const screenshotOptions = typeof screenshot !== 'string' ? { fullPage: screenshot.fullPage, omitBackground: screenshot.omitBackground } : undefined;
```

Reply:
Added thanks!